pkg/ekcoops/operator.go

```diff
@@ -92,6 +95,10 @@ func (o *Operator) Reconcile(nodes []corev1.Node, doFullReconcile bool) error {
 		return errors.Wrap(err, "failed to reconcile prometheus")
 	}
 
+	if err := o.reconcileCertificateSigningRequests(); err != nil {
+		return errors.Wrap(err, "failed to reconcile csrs")
```

Comment:
can you follow the pattern in this project and omit "failed to" from your error messages?

```
return errors.Wrap(err, "reconcile csrs")
```